gamestonk_terminal/cryptocurrency/defi/smartstake_view.py

```diff
@@ -0,0 +1,109 @@
+"""SentimentInvestor View"""
+__docformat__ = "numpy"
+
+import os
+import matplotlib.dates as mdates
+from matplotlib import pyplot as plt
+
+from gamestonk_terminal.cryptocurrency.defi import smartstake_model
+from gamestonk_terminal.helper_funcs import export_data, print_rich_table
+from gamestonk_terminal import feature_flags as gtff
+
+# pylint: disable=E1101
+
+LUNA_CIR_SUPPLY_CHANGE = "lunaSupplyChallengeStats"
+
+
+def display_luna_circ_supply_change(
+    days: int,
+    export: str,
+    supply_type: str = LUNA_CIR_SUPPLY_CHANGE,
+    limit: int = 5,
+):
+    """Display Luna circulating supply stats
+
+    Parameters
+    ----------
+    days: int
+        Number of days
+    supply_type: str
+        Supply type to unpack json
+    export: str
+        Export type
+    limit: int
+        Number of results display on the terminal
+        Default: 5
+    Returns
+        None
+    -------
+    """
+
+    df = smartstake_model.get_luna_supply_stats(supply_type, days)
+
+    if df.empty:
+        print("Error in SmartStake request")
+    else:
+        _, ax1 = plt.subplots(figsize=(25, 7))
```

Comment:
I don't know if it's only on my laptop. But for me a size of chart is to little big, I need to resize it. But it's just minor thing, if it's only my case forget about it :D

Reply:
I have a big-ass monitor so it might happen that the size is okay. I'll check with my laptop screen

Reply:
Yea that could be the case :D Let me know if it's also to big on you laptop.